src/qt/paymentserver.cpp

```diff
@@ -621,29 +622,24 @@ void PaymentServer::fetchPaymentACK(CWallet* wallet, const SendCoinsRecipient& r
     payment.add_transactions(transaction.data(), transaction.size());
 
     // Create a new refund address, or re-use:
-    QString account = tr("Refund from %1").arg(recipient.authenticatedMerchant);
-    std::string strAccount = account.toStdString();
-    std::set<CTxDestination> refundAddresses = wallet->GetAccountAddresses(strAccount);
-    if (!refundAddresses.empty()) {
-        CScript s = GetScriptForDestination(*refundAddresses.begin());
+    CPubKey newKey;
+    if (walletModel->wallet().getKeyFromPool(false /* internal */, newKey)) {
+        // BIP70 requests encode the scriptPubKey directly, so we are not restricted to address
```

Comment:
Hm yeah i think the reason i added this was just because it was a conflict there but this part looked more reasonable + it prevented the need to create an interface call for `GetAccountAddresses` + it leads to less conflicts in the future this way. So we can leave it as it is? How would you mention the partial backport in this case? Like in the code, PR title or what?

Reply:
See 36cd11f

Reply:
please change the name of the commit to start with "Partial Backport #XXXX"